### ISSUE TITLE
Test 08-Reversals/08: Rolling Takedown copia el valor de fortitude, no de damage

### ISSUE THREAD

**Issue by burgosibarra** [RodrigoToroIcarte/IIC2113-2023-1#65]
Según entiendo de la descripción, "Can only reverse a Grapple that does 7D or less. End your opponent's turn. # = D of maneuver card being reversed. Read as 0 when in your Ring area.", la carta debería copiar el valor de daño de "Gut Buster", que es 5, por lo que HHH recibiría 5 de daño
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/53795389/b1df9d1a-3950-4bf4-9260-8683cc76e8cd)

**Comment by RodrigoToroIcarte** [comment]
No, lo que pasa es que la habilidad de Mankind resta uno de daño a la maniobra. Entonces el daño neto es 4D. Mankind la revierte y por ende HHH recibe 4D (y no 5D).

**Comment by burgosibarra** [comment]
Comprendo, gracias!
